### PR TITLE
chore: improve graphql dev command

### PR DESCRIPTION
**Pull request by pedronauck**
Now after this pull request, in order to have codegen working smoothly just need to run `pnpm dev` inside `packages/graphql` folder, before it was very complicated to get codegen working properly after change any stuff inside the graphql package

### CODE REVIEW

File: packages/graphql/src/bin.ts [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
+
 config();
 
 import app from './server';
 
 // Start the server:
-app.listen(4444, () =>
-  console.log('🚀 Explorer api running at http://localhost:4444/graphql'),
-);
+app.listen(4444);
```
Comment (luizstacio) [inline packages/graphql/src/bin.ts:8]: Why remove the log here?

File: packages/graphql/tsup.config.mjs [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
 const graphqlLoaderPlugin = graphqlLoaderPluginPkg.default;
-const defConfig = {
+
+export default defineConfig((options) => ({
   outDir: 'dist',
   splitting: true,
   format: ['esm', 'cjs'],
   sourcemap: true,
-  clean: true,
+  clean: false,
```
Comment (luizstacio) [inline packages/graphql/tsup.config.mjs:13]: Why not clean the files?

**Review by luizstacio — CHANGES_REQUESTED** [review]

File: packages/graphql/tsup.config.mjs (hunk 2 above)
Comment (pedronauck) [reply packages/graphql/tsup.config.mjs:13]: because this was generating errors in the watch mode

File: packages/graphql/src/bin.ts (hunk 1 above)
Comment (pedronauck) [reply packages/graphql/src/bin.ts:8]: because I changed it to show just after the codegen

**Review by LuizAsFight — APPROVED** [review]

**Review by luizstacio — APPROVED** [review]

**Pull request merged**